src/components/Accordion/index.tsx

```diff
@@ -60,6 +60,8 @@ interface InternalAccordionProps {
 }
 
 export type AccordionProps = InternalAccordionProps & RenditionSystemProps;
+
+/** [View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Accordion/Accordion.stories.tsx) */
```

Comment:
You can actually see the code in the storybook docs page, so this is unnecessary.